docs/output_fields.rst

```diff
@@ -192,7 +192,8 @@ CPT measurements (sonderingen)
     apparaat,1,string,200kN - RUPS
     datum_gw_meting,10,datetime,2002-07-04 13:50:00
     diepte_gw_m,10,float,1.2
-    z,10,float,1.2
+    lengte,10,float,1.2
+	diepte,10,float,1.2
```

Comment:
Why the indentation? :)

Reply:
I changed it with notepad++ and is seems lengte has some spaces in front and diepte a tab.